src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -1536,6 +1536,8 @@ async function containerBuildAndRun(event: string, buildInfo: BuildRequest, oper
             logger.logProjectError(err, buildInfo.projectID, buildInfo.containerName);
             await projectStatusController.updateProjectStatus(STATE_TYPES.buildState, buildInfo.projectID, BuildState.failed, "buildscripts.buildFail");
             await projectStatusController.updateProjectStatus(STATE_TYPES.appState, buildInfo.projectID, AppState.stopped, " ");
+            const containerName = await getContainerName(projectInfo);
+            await kubeutil.deleteHelmRelease(buildInfo.projectID, containerName);
```

Comment:
@yangcao77 What happens if the helm release doesn't exist yet when the build fails? Will it prevent the error message message specified above from getting thrown?

Reply:
Just tested on che. Enable a project with invalid Dockerfile (no previous helm release exist) can still see the build error.